src/syntax/instrumentPpx.ml

```diff
@@ -65,9 +90,13 @@ let marker file ofs kind marked =
    unmodified if the expression is already marked, has a ghost location,
    construct instrumentation is disabled, or a special comments indicates to
    ignore line. *)
```

Comment:
I wanted to keep changes to a minimum. Perhaps we can have a comment cleanup change list at some point?